server/views/pages/article.njk

```diff
@@ -7,16 +7,16 @@
 {% endblock %}
 
 {% block body %}
-{% component 'page-description', { title: article.headline, meta: { type: 'date', value: article.datePublished | formatDate }, modifiers: ['a'], lead: true } %}
+
+{% componentV2 'page-description', {title: article.headline, meta: {type: 'date', value: article.datePublished | formatDate}, lead: true}, {'a': true}, {icon: article.contentType | getIconForContentType} %}
 
 {% if article.series[0] and article.series[0].commissionedLength %}
   {% set asyncContentEndpoint = '/series-nav/' + article.series[0].url + '?current=' + article.url %}
   {% component 'async', { endpoint: asyncContentEndpoint, placeholderClassname: 'series-nav-placeholder' } %}
 {% endif %}
 
-
 {% for media in article.mainMedia %}
-  {% if (media.contentUrl) %}
+  {% if (media.contentUrl) or (media.embedUrl) %}
```

Comment:
This is a hard to decipher conditional - but admittedly it was me that put the first one in.
What instances do we have where we have main media without these?